components/specialized-parser/terraform/build.tf

```diff
@@ -14,33 +14,27 @@
 
 
 locals {
-  classifier_cloud_build_content_hash = sha512(
-    join("", [
-      for f in fileset(path.module, "../src/**") :
-      filesha512("${path.module}/${f}")
-      ]
-    )
-  )
-  image_name_and_tag = "${var.region}-docker.pkg.dev/${var.project_id}/${var.artifact_repo}/processing-form-parser:latest"
+  cloud_build_fileset      = fileset(path.module, "../src/**/*")
+  cloud_build_content_hash = sha512(join("", [for f in local.cloud_build_fileset : filesha512("${path.module}/${f}")]))
+  service_account_name     = var.specialized_parser_cloud_run_job_name
+  image_name_and_tag       = "${var.region}-docker.pkg.dev/${var.project_id}/${var.artifact_repo}/${var.specialized_parser_cloud_run_job_name}:latest"
 }
 
 # See github.com/terraform-google-modules/terraform-google-gcloud
-module "gcloud_processing_form_parser" {
+module "gcloud" {
```

Comment:
nit: we have quite a few modules named "gcloud_build_something" that get referenced elsewhere, for convention and readability how about calling this  module "gcloud_build_specialized_processor"

Reply:
Done - thanks